src/tests/unittests/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../..)
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/../..)
+
+find_package(cmocka 1.1.0 CONFIG REQUIRED)
+include(AddCMockaTest)
+include(AddCMockaMockTest)
+
+add_cmocka_mock_test(simplesample)
```

Comment:
This is a comment for the commit message.

If you want to run the tests, do:

    make test

or

    ctest --output-on-failure

In the build directory!

Reply:
Hmm, this doesn't work on my machine. I'll wait until you find time to look into it (as discussed on IRC).